lib/uwb/UwbPeer.cxx

```diff
@@ -32,6 +33,47 @@ UwbPeer::UwbPeer(UwbMacAddress address) :
     m_address(std::move(address))
 {}
 
+/**
+ * @brief Assuming the Arm definition of Qm.n formatting, the most significant bit is the sign, the next
+ * (m-1) bits are an integer, and the next n bits is the number to be multiplied by pow(2,n)
+ * The double equivalent will be the sum of those two results
+ * TODO double check this conversion, write tests for it
+ *
+ * @param q97 a number in Q9.7 format
+ * @return double
+ */
+double
+ConvertQ9_7FormatToIEEE(uint16_t q97)
```

Comment:
Probably this should go into some kind of helper class or library, but we can leave it here for now (so feel free to resolve this).

Reply:
Agreed.